lib/tmuxinator/config.rb

```diff
@@ -62,7 +62,19 @@ def configs
         end
       end
 
-      def validate(name, options={})
+      def parse_settings(args)
+        settings = args.select { |x| x.match(/.*=.*/) }
+        args.reject! { |x| x.match(/.*=.*/) }
+
+        settings.map! do |setting|
+          parts = setting.split('=')
+          [parts[0], parts[1]]
+        end
+
+        Hash[*(settings.flatten)]
+      end
+
+      def validate(name, options={}, args)
```

Comment:
Assignment Branch Condition size for validate is too high. [19.82/15]<br>Surrounding space missing in default value assignment.